src/test/java/org/gridsuite/explore/server/ExploreTest.java

```diff
@@ -168,6 +172,15 @@ public MockResponse dispatch(RecordedRequest request) throws InterruptedExceptio
                     return new MockResponse().setResponseCode(200);
                 } else if (path.matches("/v1/filters/.*/replace-with-script") && "PUT".equals(request.getMethod())) {
                     return new MockResponse().setResponseCode(200);
+                } else if ("DELETE".equals(request.getMethod())) {
```

Comment:
Why we do not mock the bean service instead ?

Reply:
need to test complete metadata